src/components/__tests__/InstantSearchSsr.js

```diff
@@ -98,33 +98,39 @@ it('does not dispose if not yet started', async () => {
 
   const disposeSpy = jest.spyOn(instance, 'dispose');
 
-  const wrapper = mount(InstantSearchSsr, {
+  const wrapper = mount({
     provide: {
       $_ais_ssrInstantSearchInstance: instance,
     },
     components: {
-      AisSearchBox: SearchBox,
-    },
-    slots: {
-      default: SearchBox,
+      InstantSearchSsr,
+      SearchBox,
     },
+    template: `
+      <InstantSearchSsr>
+        <SearchBox />
+      </InstantSearchSsr>
+    `,
   });
 
   wrapper.destroy();
 
   // does not yet call, since instance isn't started
   expect(disposeSpy).toHaveBeenCalledTimes(0);
 
-  const wrapperTwo = mount(InstantSearchSsr, {
+  const wrapperTwo = mount({
     provide: {
       $_ais_ssrInstantSearchInstance: instance,
     },
     components: {
-      AisSearchBox: SearchBox,
-    },
-    slots: {
-      default: SearchBox,
+      InstantSearchSsr,
+      SearchBox,
     },
+    template: `
+      <InstantSearchSsr>
```

Comment:
since all of these tests have been updated, does __Template-test.js also need to be updated?

Reply:
Not sure what you have in mind, but there seems not much to do about the file.
https://github.com/algolia/vue-instantsearch/blob/a43c889402532f9bfda816fed1ffc45a9225781d/src/components/__tests__/__Template.js#L10:L10

Reply:
unlike the other tests, the template still mounts the component directly, which means it would be different with slots, should it also use template?

Reply:
Oh I get what you mean. But there are still lots of tests mounting component directly when there's nothing to do with slots. Do you see it as inconsistency?

![image](https://user-images.githubusercontent.com/499898/126297177-8899a180-b0dc-48aa-bb79-abf8a23f90a7.png)